packages/databyss-api/src/routes/api/helpers/pagesHelper.js

```diff
@@ -9,14 +10,17 @@ const getBlockItemsFromId = blocks => {
       _id,
     }).catch(err => console.log(err))
     if (block) {
-      const { type, entryId, sourceId, authorId } = block
+      const { type, entryId, sourceId, authorId, topicId } = block
       const response = { type, _id }
       if (type === 'ENTRY') {
```

Comment:
a more functional way to do this would be:
```
response.refId = {
  ENTRY: entryId,
  SOURCE: sourceId,
  TOPIC: topicId,
  AUTHOR: authorId,
}[type]
```